### PR TITLE
replace reverse_geocode with geocoder_reverse_natural_earth

### LINKED ISSUE #898: Errors in Country Database
The following countries are problematic in the pyaerocom country database:

MK: Macedonia -> North Macedonia
TR: Turkey -> Türkiye

SJ: Svalbard and Jan Mayen should be part of NO

### PR DESCRIPTION
**Pull request by jgriesfeller**
1st ry to pass CI, not entirely "nice" yet

### CODE REVIEW

**Comment by jgriesfeller** [comment]
CI is not passing due to an unrelated test failure. Annoying.

**Comment by jgriesfeller** [comment]
the countries determined by `reverse_geocode` and `geocoder_reverse_natural_earth` are somewhat different (from CI)
```
>       assert countries == [
            "Italy",
            "Japan",
            "Burkina Faso",
            "Brazil",
            "American Samoa",
            "French Southern Territories",
            "Korea, Republic of",
            "France",
            "Portugal",
            "France",
            "Barbados",
            "United Kingdom",
            "Bolivia",
            "United States",
            "French Polynesia",
            "China",
            "Taiwan",
            "Algeria",
            "Netherlands",
            "Greece",
            "Belgium",
            "Argentina",
        ]
E       AssertionError: assert ['Norway', 'Japan', 'Mali', 'Brazil', 'New Zealand', 'New Zealand', 'Republic of Korea', 'France', 'France', 'France', 'Barbados', 'Norway', 'Bolivia', 'United States', 'Fr. Polynesia', 'China', 'Taiwan', 'Algeria', 'Netherlands', 'Greece', 'Norway', 'Argentina'] == ['Italy', 'Japan', 'Burkina Faso', 'Brazil', 'American Samoa', 'French Southern Territories', 'Korea, Republic of', 'France', 'Portugal', 'France', 'Barbados', 'United Kingdom', 'Bolivia', 'United States', 'French Polynesia', 'China', 'Taiwan', 'Algeria', 'Netherlands', 'Greece', 'Belgium', 'Argentina']
E         
E         At index 0 diff: 'Norway' != 'Italy'
E         
E         Full diff:
E           [
E         -     'Italy',
E         +     'Norway',
E               'Japan',
E         -     'Burkina Faso',
E         +     'Mali',
E               'Brazil',
E         -     'American Samoa',
E         -     'French Southern Territories',
E         -     'Korea, Republic of',
E         +     'New Zealand',
E         +     'New Zealand',
E         +     'Republic of Korea',
E               'France',
E         -     'Portugal',
E         +     'France',
E               'France',
E               'Barbados',
E         -     'United Kingdom',
E         +     'Norway',
E               'Bolivia',
E               'United States',
E         -     'French Polynesia',
E         ?        ^^^^
E         +     'Fr. Polynesia',
E         ?        ^
E               'China',
E               'Taiwan',
E               'Algeria',
E               'Netherlands',
E               'Greece',
E         -     'Belgium',
E         +     'Norway',
E               'Argentina',
E           ]
```

**Comment by jgriesfeller** [comment]
It turns out the country detection is not always correct:
```
print(geo.lookup_nearest(45.31390, 12.508300)["ISO_A2_EH"])
NO
```
but the actual location is somewhere around Venice (Italy; screenshot of Aeronet's own map):
image
Google maps:
image
Any ideas @heikoklein ?

**Comment by lewisblake** [comment]
Blocked by https://github.com/metno/geocoder_reverse_natural_earth/issues/3

**Comment by jgriesfeller** [comment]
All tests besides `tests/aeroval/test_helpers.py::test_make_dummy_model[cfgexp1] ` pass now. Since we have ignored that for some other PRs as well, please review and merge before releasing the new version

File: tests/plugins/gaw/test_dms.py [hunk 1]
```diff
@@ -58,12 +83,12 @@ def test_vmrdms_ams(data_vmrdms_ams_cvo):
     vmrdms = stat["vmrdms"]
     assert_allclose(
         [vmrdms.mean(), vmrdms.std(), vmrdms.max(), vmrdms.min()],
-        [185.6800736155262, 237.1293922258991, 2807.6, 5.1],
+        [1.856801e14, 2.371294e14, 2.807600e15, 5.100000e12],
```
Comment (lewisblake) [inline tests/plugins/gaw/test_dms.py:86]: This are all changes by an order of magnitude of 10^12. Just want to make sure you're aware of this and there is a reason for it.

**Review by lewisblake — APPROVED** [review]
Looks good. Curious about the change in the order of magnitude in the tests but we can discuss why in person.

File: tests/plugins/gaw/test_dms.py (hunk 1 above)
Comment (jgriesfeller) [reply tests/plugins/gaw/test_dms.py:86]: Units, I guessed.

**Pull request merged**